src/core/database/query.helpers.ts

```diff
@@ -1102,3 +1103,19 @@ export function getPropList(query: Query, ...aliases: string[]) {
     ])
     .with(['collect(prop) as propList', 'node', ...aliases]);
 }
+
+//List filtering after refactor
+export function applyListFilters<
```

Comment:
Let's not do this. Each filter property can have different handling so handling as a complete object will not work.

Reply:
Removed 😢